internal/gcsx/storage_handle.go

```diff
@@ -0,0 +1,68 @@
+package gcsx
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/option"
+)
+
+type storageHandle struct {
+	client *storage.Client
+}
+
+type storageClientConfig struct {
+	disableHttp2        bool
+	maxConnsPerHost     int
+	maxIdleConnsPerHost int
+	tokenSrc            oauth2.TokenSource
+}
+
+// GetStorageClientHandle returns the handle of Go storage client containing
+// customized http client. We can configure the http client using the
+// storageClientConfig parameter.
+func GetStorageClientHandle(ctx context.Context,
+	sc storageClientConfig) (sh *storageHandle, err error) {
+	var storageClient *storage.Client
+
+	var tr *http.Transport
+	// Choosing between HTTP1 and HTTP2. HTTP2 is more performant.
```

Comment:
in our case http1 is more performant. It would be good to elaborate what do you mean by http2 is more performant like scenarios

Reply:
This was by mistake, wanted to write disabling http2 make the client more performant.